<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="91446" started-at="2022-05-05T09:23:02Z" finished-at="2022-05-05T09:24:33Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="91446" started-at="2022-05-05T09:23:02Z" finished-at="2022-05-05T09:24:33Z">
      <class name="com.VTiger.TestCases.TC01_CreateOrg">
        <test-method status="PASS" signature="connecttoDB()[pri:0, instance:com.VTiger.TestCases.TC01_CreateOrg@47d90b9e]" name="connecttoDB" is-config="true" duration-ms="53" started-at="2022-05-05T09:23:02Z" finished-at="2022-05-05T09:23:02Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- connecttoDB -->
        <test-method status="PASS" signature="launchBrowser()[pri:0, instance:com.VTiger.TestCases.TC01_CreateOrg@47d90b9e]" name="launchBrowser" is-config="true" duration-ms="25590" started-at="2022-05-05T09:23:02Z" finished-at="2022-05-05T09:23:28Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchBrowser -->
        <test-method status="PASS" signature="login()[pri:0, instance:com.VTiger.TestCases.TC01_CreateOrg@47d90b9e]" name="login" is-config="true" duration-ms="6058" started-at="2022-05-05T09:23:28Z" finished-at="2022-05-05T09:23:34Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method status="FAIL" signature="createorg()[pri:0, instance:com.VTiger.TestCases.TC01_CreateOrg@47d90b9e]" name="createorg" duration-ms="39146" started-at="2022-05-05T09:23:34Z" finished-at="2022-05-05T09:24:13Z">
          <exception class="junit.framework.AssertionFailedError">
            <full-stacktrace>
              <![CDATA[junit.framework.AssertionFailedError
at junit.framework.Assert.fail(Assert.java:48)
at junit.framework.Assert.assertTrue(Assert.java:20)
at junit.framework.Assert.assertFalse(Assert.java:34)
at junit.framework.Assert.assertFalse(Assert.java:41)
at com.VTiger.TestCases.TC01_CreateOrg.createorg(TC01_CreateOrg.java:49)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- junit.framework.AssertionFailedError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createorg -->
        <test-method status="PASS" signature="logout()[pri:0, instance:com.VTiger.TestCases.TC01_CreateOrg@47d90b9e]" name="logout" is-config="true" duration-ms="6179" started-at="2022-05-05T09:24:13Z" finished-at="2022-05-05T09:24:19Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logout -->
        <test-method status="PASS" signature="closeBrowser()[pri:0, instance:com.VTiger.TestCases.TC01_CreateOrg@47d90b9e]" name="closeBrowser" is-config="true" duration-ms="13941" started-at="2022-05-05T09:24:19Z" finished-at="2022-05-05T09:24:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
        <test-method status="PASS" signature="disConnectDB()[pri:0, instance:com.VTiger.TestCases.TC01_CreateOrg@47d90b9e]" name="disConnectDB" is-config="true" duration-ms="0" started-at="2022-05-05T09:24:33Z" finished-at="2022-05-05T09:24:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- disConnectDB -->
      </class> <!-- com.VTiger.TestCases.TC01_CreateOrg -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
